CARDS-2665: Add support for configuring the precision of decimal/double answers

marta- opened the pull request:
To test:
* create/use a questionnaire with decimal and double questions
* from the questionnaire wizard set the Decimal scale property of the questions to a value such as 1 or 2
* test that only that many decimals can be entered in the answers
OR
* test in the `corpath` project according to the instructions from https://github.com/data-team-uhn/corpath/pull/17.

marta- committed:
CARDS-2665: Add support for configuring the precision of decimal/double answers

sdumitriu reviewed (approved):
Looks good, didn't test.

veronikaslc commented:
Tested

sdumitriu reviewed (approved):
When combining scale with slider, it is possible to have numbers with more precision than intended by the scale, for example set scale to 0 and slider tick to 0.1. I think it's OK, since this is user error, but could we maybe not show the scale option if the display mode is slider?

marta- committed:
CARDS-2665: Add support for configuring the precision of decimal/double answers

Addressing review comments: only make the decimal scale available when the display mode includes input, to avoid conflicting with slider precision settings.
